pallets/automation-time/src/migrations.rs

```diff
@@ -0,0 +1,28 @@
+use crate::{Config, Weight};
+use frame_support::traits::Get;
+use log::info;
+
+pub mod v1 {
+	use frame_support::{migration::get_storage_value, traits::StorageVersion};
+
+  use crate::{LastTimeSlot, Pallet};
+
+use super::*;
+
+	pub fn migrate<T: Config>() -> Weight {
+		info!(target: "automation-time", "Migrating automation-time v1");
+    let pallet_prefix: &[u8] = b"AutomationTime";
+		let storage_item_prefix: &[u8] = b"LastTimeSlot";
+
+		let stored_data = get_storage_value::<u64>(
+			pallet_prefix,
+			storage_item_prefix,
+      &[],
```

Comment:
idk why it pops out like that - the formatting is fine locally. also what's the command for running fmt again? 😅 

and that's supposed to be the key hash, but since we're just storing a single value without any hash, we give it empty array. https://docs.substrate.io/rustdocs/latest/frame_support/storage/migration/fn.get_storage_value.html